src/pages/odklad/suhrn.tsx

```diff
@@ -17,6 +17,9 @@ const Suhrn: NextPage<Props> = ({ postponeUserInput }: Props) => {
   useEffect(() => {
     router.prefetch(nextRoute);
   });
+  const [firstName, ...lastNames] = postponeUserInput.meno_priezvisko
```

Comment:
Ja som sem dal zatial `meno_priezvisko` ako placeholde (a po pravde potom na to zabudol 😃). Najprv som myslel ze tiez budeme mat 2 objekty (ako `TaxFormUserInput` a `TaxForm`), tj ze by sme mali `Postpone` ktory vypocita meno a priezvisko z, `PostponeUserInput`.

Ak to nepotrebjeme (len kvoli menu a prievisku to netreba), tak mozno dat niekam do utils metodu `parseFullName` ktora vrati meno a priezvisko z celeho mena.

Reply:
Hej, kedze tu vlastne netreba nic pocitat tak jednoduchsie bolo myslim mapovat rovno `PostponeUserInput` na `PostponeOutput`. 

Podla mna je to fajn ze meno_priezvisko je spolu, lepsie sa to pouziva s tym autoformom takto. A hej, tu funkciu mozeme neskor spravit, zatial copy/paste je king :D